policyengine_us/variables/gov/states/mi/tax/income/credits/homestead_property_tax_credit/mi_homestead_property_tax_credit_non_refundable.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_us.model_api import *
+
+
+class mi_homestead_property_tax_credit_non_refundable(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan non-refundable Homestead Property Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mi.tax.income.credits.homestead_property_tax_credit
+
+        total_household_resources = tax_unit("mi_household_resources", period)
+
+        # disabled
+        disabled_people = add(tax_unit, period, ["is_disabled"])
```

Comment:
use a select to determine whether the filer is (a) aged, (b) disabled, or (c) aged and disabled and return (a) calculation mi_homestead_property_tax_credit_refundable with the senior phase out or (b) and (c) just the mi_homestead_property_tax_credit_refundable

Reply:
4th condition - not aged and not disabled - 4th output mi_homestead_property_tax_credit_refundable * 0.6